src/get-started/java-binding.md

```diff
@@ -1000,17 +1002,17 @@ exonum-java run \
     --private-api-address 127.0.0.1:3010
 ```
 
-#### Changing the used JVM
+#### Changing the Used JVM
 
-By default, Exonum Java App automatically finds system JVM and uses it. To
-change this, the user can set `JAVA_HOME` environment variable pointing at the
+By default, Exonum Java App automatically finds the system JVM and uses it. To
+change this, set the `JAVA_HOME` environment variable pointing at the
```

Comment:
?
set the `JAVA_HOME` environment variable to the JVM installation directory.

Reply:
Why do you think the current one is invalid? I can find multiple usages of `variable pointing` in the Internet.

Reply:
I am not saying 'invalid', but "set a variable to a value" is shorter. @Maria-Nosyk might advise 9n what is clearer.